src/Grid/Grid.jsx

```diff
@@ -0,0 +1,59 @@
+import classnames from "classnames";
+import React, {PropTypes} from "react";
+
+import Col from "./Col";
+import Row from "./Row";
+import Size from "./Size";
+
+require("../less/grid.less");
+
+
+export function Grid({
+  className,
+  children,
+  wrapperComponent: Wrapper,
+  ...additionalProps,
+}) {
+  const {cssClass} = Grid;
+
+  return (
+    <Wrapper className={classnames(cssClass.GRID, className)} {...additionalProps}>
+      {children}
+    </Wrapper>
+  );
+}
+
+Grid.Row = Row;
+Grid.Col = Col;
```

Comment:
wanted `Grid` to be a consistent entry point, so clients can use `<Grid />`, `<Grid.Row />` and `<Grid.Col />` instead of `<Grid.Row.Col />`